specs-go/config.go

```diff
@@ -108,6 +108,14 @@ type Platform struct {
 	OS string `json:"os"`
 	// Arch is the architecture
 	Arch string `json:"arch"`
+	// OSVersion specifies the operating system version, for example `10.0.10586`.
```

Comment:
> I'd pick a more up to date build.

My take on the discussion is that [runtime-spec will likely drop `platform`][1] or [keep only `platform.os`][2].  Either way, this line is unlikely to land in runtime-spec, so we probably don't need to worry about it.  And the image-spec equivalent is getting , so I doubt we need to worry about this line.

And the image-spec analog got a quad-touble in the recently-landed opencontainers/image-spec#651.

[1]: https://github.com/opencontainers/runtime-spec/pull/830#issuecomment-303463569
[2]: https://github.com/opencontainers/runtime-spec/pull/830#issuecomment-303426610